src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -344,7 +344,8 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
         </>
       ) : pia?.status === PiaStatuses.EDIT_IN_PROGRESS ||
         pia?.status === PiaStatuses.INCOMPLETE ||
-        pia?.status === PiaStatuses.MPO_REVIEW ? (
+        pia?.status === PiaStatuses.MPO_REVIEW ||
```

Comment:
What is this change doing?

Reply:
Agree.  All status checks should now be taken from the state table and not have checks like this. Part of future refactor.

Reply:
> What is this change doing?

I added it so that when the PIA status is in CPO status, it now displays the "Pending Review" status in the review section.

Reply:
> Agree. All status checks should now be taken from the state table and not have checks like this. Part of future refactor.

yes, I saw that we're still checking statuses in a similar way in multiple places. It seems like we should change things around, so that we get these status checks from a state table.